installFetch moved/renamed in sveltekit>=341

Builds on the latest sveltekit release return this error:

`.svelte-kit/appengine-tmp/entry.js:2:9: error: No matching export in "node_modules/@sveltejs/kit/dist/node/polyfills.js" for import "installFetch"`

[Sveltekit 341](https://github.com/sveltejs/kit/releases/tag/%40sveltejs%2Fkit%401.0.0-next.341) renamed and moved `installFetch`. I think the fix is just to use `installPolyfills` from `@sveltejs/kit/node/polyfills` instead of `installFetch`.

--- a/files/entry.js
+++ b/files/entry.js
@@ -1,5 +1,5 @@
 import path from 'node:path';
-import {installFetch} from '@sveltejs/kit/install-fetch';
+import {installPolyfills} from '@sveltejs/kit/node/polyfills';
 import {getRequest, setResponse} from '@sveltejs/kit/node';
 import compression from 'compression';
 import {manifest} from 'MANIFEST';
@@ -7,7 +7,7 @@ import polka from 'polka';
 import sirv from 'sirv';
 import {Server} from 'SERVER';
 
-installFetch();
+installPolyfills();
 
 const app = new Server(manifest);
 

--- a/package.json
+++ b/package.json
@@ -35,7 +35,7 @@
     "files"
   ],
   "peerDependencies": {
-    "@sveltejs/kit": ">=1.0.0-next.280"
+    "@sveltejs/kit": ">=1.0.0-next.341"
   },
   "devDependencies": {
     "chai": "^4.3.4",

